docs/core/porting/third-party-deps.md

```diff
@@ -1,38 +1,35 @@
 ---
-title: Porting to .NET Core - Analyzing your Third-Party Party Dependencies
-description: Porting to .NET Core - Analyzing your Third-Party Dependencies
-keywords: .NET, .NET Core
+title: Porting to .NET Core - Analyzing your third-party dependencies
+description: Learn how to analyze third-party dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 08/08/2017
 ms.topic: article
 ms.prod: .net-core
 ms.devlang: dotnet
 ms.assetid: b446e9e0-72f6-48f6-92c6-70ad0ce3f86a
 ---
+# Porting your code to .NET Core - Analyzing your third-party dependencies
 
-# Porting to .NET Core - Analyzing your Third-Party Party Dependencies
-
-The first step in the porting process is to understand your third party dependencies.  You need to figure out which of them, if any, don't yet run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+If you're looking to port your code to .NET Core, the first step in the porting process is to understand your third-party dependencies. You need to evaluate which of them, if any, don't yet run on .NET Core and develop a contingency plan for these.
 
 ## Prerequisites
 
-This article will assume you are using Windows and Visual Studio, and that you have code which runs on the .NET Framework today.
+This article assumes you're using Windows and Visual Studio, and that you have code that currently runs on the .NET Framework.
```

Comment:
It seems a bit weird to mention Windows here because it makes it sound we're only talking about how you can port to .NET Core running on Windows. We really mean that we assume the customer is *starting* on Windows with .NET Framework. The end result might run on Linux.